foundation/foundation-mda/src/main/resources/templates/deployment/argocd/airflow.yaml.vm

```diff
@@ -1,6 +1,8 @@
 apiVersion: argoproj.io/v1alpha1
 kind: Application
 metadata:
+  annotations:
+    argocd.argoproj.io/sync-wave: "1"
```

Comment:
A: This shouldn't be necessary since we're setting config store to `-2` and the default is 0.